docs/trusted_types.rst

```diff
@@ -0,0 +1,137 @@
+===================================
+Implementing Trusted Types with CSP
+===================================
+
+``DOM Cross-site Scripting``
+============================
+Cross-site scripting (XSS) is one of the most prevalent vulnerabilities
+on the web. Nonce-based CSP is used to prevent server-side XSS. Trusted
+Types are used to prevent client-side or DOM-XSS_. Trusted Types rely on
+the browser to enforce the policy that is provided to it. Currently, Trusted
+Types are supported on Chrome 83 and Android Webview. Many browsers are in the
+process of adding support. Check back for updated compatibility_.
```

Comment:
nit: would be good to mention the polyfill too https://github.com/w3c/webappsec-trusted-types#polyfill